### PR TITLE
Separate nft and eth class & Add getNfts functions in nft class

### PR DESCRIPTION
**Pull request by woomurf**
- Separate nft and eth class
  - nft class will support ainft features.
  - eth class will support features about eth contracts and nfts.
  - The reason I separated this class is that AINFT is the main feature of AinftJs and it is unlikely that nft support from other chains will be expanded any more.

### CODE REVIEW

File: src/eth.ts [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+   * @returns
+   */
+  getAppNftSymbolList({ appId }: GetAppNftSymbolListParams): Promise<string[]> {
+    const query = { appId };
+    const trailingUrl = 'symbol';
+    return this.sendRequest(HttpMethod.GET, trailingUrl, query);
+  }
+
+  /**
+   * Remove nft symbol in app.
```
Comment (platfowner) [inline src/eth.ts:55]: minor) 'from app'?

**Review by platfowner — APPROVED** [review]
LGTM with some minor comments.
Thanks!

File: src/eth.ts (hunk 1 above)
Comment (woomurf) [reply src/eth.ts]: fixed. Thanks!

**Comment by woomurf** [comment]
Thanks for the review! Now merged! @platfowner

**Pull request merged**